src/unit.js

```diff
@@ -0,0 +1,38 @@
+define([
+	"./core",
+	"./unit/format",
+
+  "./plural"
+], function( Globalize, unitFormat ) {
+
+/**
+ * Globalize.formatUnit( value, unit, options )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ */
+Globalize.prototype.formatUnit = function( value, unit, options ) {
```

Comment:
Let's update this according to the below please. So, it's accessible via both static function (`Globalize.formatUnit`) and instance method (new `Globalize(locale).formatUnit`)

```js
Globalize.formatUnit =
Globalize.prototype.formatUnit = function( ...
```